src/CodeGen_OpenCL_Dev.cpp

```diff
@@ -1183,11 +1184,15 @@ void CodeGen_OpenCL_Dev::init_module() {
     }
 
     if (target.has_feature(Target::CLHalf)) {
+        constexpr unsigned short nan_f16 = float16_t::exponent_mask | float16_t::mantissa_mask;
+        constexpr unsigned short neg_inf_f16 = float16_t::sign_mask | float16_t::exponent_mask;
+        constexpr unsigned short inf_f16 = float16_t::exponent_mask;
```

Comment:
Nit: why not use `float16_t::make_nan()` et al?

Reply:
Because I didn't see that when I briefly scrolled through the class 🙂

Reply:
Ah, the `make_*` functions aren't constexpr

Reply:
I'm using them anyway; thanks for the catch.